cmd/iam-object-store.go

```diff
@@ -448,7 +448,7 @@ func (iamOS *IAMObjectStore) listAllIAMConfigItems(ctx context.Context) (map[str
 			}
 		}
 
-		if !found && !(item.Item == "config/config.json" || item.Item == "/format.json" || strings.Contains(item.Item, "config/history/")) {
+		if !found {
```

Comment:
We still need to ignore format.json here (`&& item.item != "format.json"`)

Tested otherwise

Reply:
done